Extension/package.json

```diff
@@ -1624,7 +1624,7 @@
   },
   "scripts": {
     "vscode:prepublish": "yarn run compile",
-    "compile": "yarn run precompile && webpack --mode production --vscode-nls",
+    "compile": "yarn run precompile && yarn run translations-generate && webpack --mode production --vscode-nls",
     "compile-dev": "yarn run precompile && webpack --mode development",
     "compile-watch": "yarn run precompile && webpack --mode production --vscode-nls --watch --info-verbosity verbose",
```

Comment:
Sorry, I gave you bad feedback on this.  Obviously, 'compile-watch' is for production and 'compile-dev-watch' is not.  Could you put the 'yarn run translations-generate' back on 'compile-watch'.   :]

Reply:
Sure